src/main/cpp/commands/ExampleCommand.cpp

```diff
@@ -4,5 +4,4 @@
 
 #include "commands/ExampleCommand.h"
 
-ExampleCommand::ExampleCommand(ExampleSubsystem* subsystem)
-    : m_subsystem{subsystem} {}
+ExampleCommand::ExampleCommand() {}
```

Comment:
why modify the example command?

Reply:
It causes errors if I don't have the ExampleSubsystem